extensions/amp-list/0.1/test/integration/test-amp-list.js

```diff
@@ -27,7 +27,7 @@ const body =
 
 const extensions = ['amp-list', 'amp-mustache', 'amp-bind'];
 
-describe('amp-list', function() {
+describe.configure().run('amp-list', function() {
```

Comment:
Yeah, wrapping it in `run('...` is required to automatically skip the dev browsers. We could refactor out `describe.configure().run('...` of all the tests in a future PR.

Reply:
btw, I think instead of skipping the dev browsers, we should run all tests in the equivalent dev version but make it so that failures in those browsers not fail the Travis build.

Eventually, when we get to #14835 (which I'm planning on making part of my Q1-Q2 goals) these failures will be noticed early and we can get them resolved before it's too late.

TL;DR: don't refactor the entire test suite :smile:

Reply:
Definitely not looking to do a big refactor :)